backups/backup.py

```diff
@@ -82,7 +82,9 @@ def run_importer_script() -> None:
         assert popen.stdout
```

Comment:
I would probably do the `popen = subprocess.Popen(...)` construct here with a `with subprocess.Popen(...) as popen: ...` as it is done in our exporter.py starting from line 131

Reply:
How do we read the `stderr` then at the end of the process, but before popen is closed?

Reply:
I think you can just add
```
if popen.stderr:
  stderr = popen.stderr.read()
```
as the last line or?

Reply:
Implemented.